proto/internal/temporal/server/api/historyservice/v1/request_response.proto

```diff
@@ -556,6 +556,7 @@ message VerifyChildExecutionCompletionRecordedRequest {
     int64 parent_initiated_id = 4;
     int64 parent_initiated_version = 5;
     temporal.server.api.clock.v1.VectorClock clock = 6;
+    bool resend_parent = 7;
```

Comment:
no strong opinion. But as this API is verifyChild. maybe we should handle the resend logic on the caller side?